src/components/AddItems.jsx

```diff
@@ -16,14 +16,32 @@ const radioInputOptions = {
 	notSoon: ['notsoon', nextPurchaseDate.notSoon, 'Not Soon'],
 };
 
-export function AddItems() {
+export function AddItems({ data }) {
```

Comment:
Props should be named to give more context. Data is vague and could mean anything, so it's important to name it as close to what it is as possible

```suggestion
export function AddItems({ items }) {
```

Reply:
I just realized these might be changes that were from the rebase. I'll duplicate this message onto the other PR

Reply:
fixed!